d3b_utils/aws_bucket_contents.py

```diff
@@ -81,9 +108,15 @@ def fetch_bucket_obj_info(
             )
             dict_writer.writeheader()
             dict_writer.writerows(bucket_contents)
+        if get_versions and delete_markers:
+            keys = set(chain(*(d.keys() for d in delete_markers)))
+            with open("delete_markers" + output_filename, "w") as f:
+                dict_writer = csv.DictWriter(
+                    f, restval="", fieldnames=keys, delimiter=delim
+                )
+                dict_writer.writeheader()
+                dict_writer.writerows(delete_markers)
 
+    if get_versions and delete_markers:
+        return "a dict of bucket_contents + delete markers maybe?"
     return bucket_contents
-
-
-# backwards compatibility
-fetch_aws_bucket_obj_info = fetch_bucket_obj_info
```

Comment:
I don't see a reason to remove this alias

Reply:
oh oops - did that by mistake. Thanks for catching that.

Reply:
fixed